drift_dev/lib/src/cli/commands/make_migrations.dart

```diff
@@ -0,0 +1,505 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:drift_dev/src/analysis/results/results.dart';
+import 'package:drift_dev/src/cli/cli.dart';
+import 'package:drift_dev/src/cli/commands/schema.dart';
+import 'package:drift_dev/src/cli/commands/schema/generate_utils.dart';
+import 'package:drift_dev/src/cli/commands/schema/steps.dart';
+
+import 'package:drift_dev/src/services/schema/schema_files.dart';
+import 'package:io/ansi.dart';
+import 'package:path/path.dart' as p;
+
+class MakeMigrationCommand extends DriftCommand {
+  MakeMigrationCommand(super.cli);
+
+  @override
+  String get description => """
+Generates migrations utilities for drift databases
+
+### Usage
```

Comment:
We can't really do markdown on the CLI. You could try making it bold, but most of this command should be explained on the website as well.